src/Engine/MapController.java

```diff
@@ -666,7 +673,7 @@ public void animationEnded( GameEvent event, boolean animEventQueueIsEmpty )
       }
 
       // If fewer than two COs yet survive, the game is over.
-      if(activeNum < 2)
+      if( activeNum < 2 )
```

Comment:
Yeah, we really need to reconcile our autoformat settings. I don't want pull requests with 90% whitespace changes.